src/main/java/core/basesyntax/interfaces/Flyable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.interfaces;
+
+public interface Flyable {
+    public void fly();
```

Comment:
```suggestion
   void fly();
```
methods in interfaces are public by default, intellij should give you a hint about it